src/Symfony/Component/Routing/README.md

```diff
@@ -3,10 +3,48 @@ Routing Component
 
 The Routing component maps an HTTP request to a set of configuration variables.
 
+Getting Started
+---------------
+
+```
+$ composer require symfony/routing
+```
+
+```php
+use App\Controller\BlogController;
+use Symfony\Component\Routing\Generator\UrlGenerator;
+use Symfony\Component\Routing\Matcher\UrlMatcher;
+use Symfony\Component\Routing\RequestContext;
+use Symfony\Component\Routing\Route;
+use Symfony\Component\Routing\RouteCollection;
+
+$route = new Route('/blog/{slug}', ['_controller' => BlogController::class]);
+$routes = new RouteCollection();
+$routes->add('blog_show', $route);
+
+$context = new RequestContext('/');
```

Comment:
the base url param `/` does not seem necessary and could just confuse people. I'd suggest to remove it.

Reply:
Thanks, I've removed it :)